algorithms/benchmark/bruteforce.py

```diff
@@ -0,0 +1,47 @@
+from dist_sqr import dist_sqr
+
+
+def bruteforce(points: list[list], search_radius: float, epsilon: float) -> list[list]:
```

Comment:
Let's create a class for each algorithm. Such params as search radius and epsilon could be set in the ctor.

Reply:
Something like
```python
class BruteForce:
   def __init__(self, search_radius, search_radius):
       self.search_radius = search_radius
       self.search_radius = search_radius
   
   def search(self, points):
       pass
```